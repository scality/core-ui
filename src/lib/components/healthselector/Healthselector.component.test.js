import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  configPath: 'src/lib/components/healthselector/__snapshots__',
});
