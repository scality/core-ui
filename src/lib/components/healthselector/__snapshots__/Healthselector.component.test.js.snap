// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Healthselector Default 1`] = `
<div
  className="sc-cSHVUG dZiVUo"
>
  <h3
    className="sc-kAzzGY cVbXJ"
  >
    Basic usage
  </h3>
  <div
    className="sc-htoDjs ixiQr sc-healthselector"
  >
    <button
      className="sc-bxivhb sc-gqjmRU bGfYun trigger"
      onBlur={[Function]}
      onClick={[Function]}
      size="base"
    >
      <div
        className="sc-VigVT ceQhNn"
      >
        <span
          className="sc-gZMcBi bvGPfj"
        >
          <span
            className="sc-dnqmqq hPNZvv"
          />
          <span
            className="sc-dnqmqq gsxYuK"
          />
          <span
            className="sc-dnqmqq jrdsWP"
          />
        </span>
      </div>
      <div
        className="sc-jTzLTM hoEajO"
      >
        All
      </div>
      <i
        className="fas fa-caret-down"
      />
    </button>
  </div>
  <h3
    className="sc-kAzzGY cVbXJ"
  >
    Health Selector Sizes (Triggering StoryBook actions)
  </h3>
  <div
    className="sc-kGXeez httvMN"
  >
    <div
      className="sc-kpOJdX iVjlhL"
    >
      <div
        className="sc-htoDjs ixiQr sc-healthselector"
      >
        <button
          className="sc-bxivhb sc-gqjmRU fnjuum trigger"
          onBlur={[Function]}
          onClick={[Function]}
          size="smaller"
        >
          <div
            className="sc-VigVT ceQhNn"
          >
            <span
              className="sc-gZMcBi bvGPfj"
            >
              <span
                className="sc-dnqmqq hPNZvv"
              />
              <span
                className="sc-dnqmqq gsxYuK"
              />
              <span
                className="sc-dnqmqq jrdsWP"
              />
            </span>
          </div>
          <div
            className="sc-jTzLTM hoEajO"
          >
            All
          </div>
          <i
            className="fas fa-caret-down"
          />
        </button>
      </div>
    </div>
    <div
      className="sc-kpOJdX iVjlhL"
    >
      <div
        className="sc-htoDjs ixiQr sc-healthselector"
      >
        <button
          className="sc-bxivhb sc-gqjmRU eLVUCO trigger"
          onBlur={[Function]}
          onClick={[Function]}
          size="small"
        >
          <div
            className="sc-VigVT ceQhNn"
          >
            <span
              className="sc-gZMcBi bvGPfj"
            >
              <span
                className="sc-dnqmqq hPNZvv"
              />
              <span
                className="sc-dnqmqq gsxYuK"
              />
              <span
                className="sc-dnqmqq jrdsWP"
              />
            </span>
          </div>
          <div
            className="sc-jTzLTM hoEajO"
          >
            All
          </div>
          <i
            className="fas fa-caret-down"
          />
        </button>
      </div>
    </div>
    <div
      className="sc-kpOJdX iVjlhL"
    >
      <div
        className="sc-htoDjs ixiQr sc-healthselector"
      >
        <button
          className="sc-bxivhb sc-gqjmRU bGfYun trigger"
          onBlur={[Function]}
          onClick={[Function]}
          size="base"
        >
          <div
            className="sc-VigVT ceQhNn"
          >
            <span
              className="sc-gZMcBi bvGPfj"
            >
              <span
                className="sc-dnqmqq hPNZvv"
              />
              <span
                className="sc-dnqmqq gsxYuK"
              />
              <span
                className="sc-dnqmqq jrdsWP"
              />
            </span>
          </div>
          <div
            className="sc-jTzLTM hoEajO"
          >
            All
          </div>
          <i
            className="fas fa-caret-down"
          />
        </button>
      </div>
    </div>
  </div>
  <div
    className="sc-kGXeez httvMN"
  >
    <div
      className="sc-kpOJdX iVjlhL"
    >
      <div
        className="sc-htoDjs ixiQr sc-healthselector"
      >
        <button
          className="sc-bxivhb sc-gqjmRU kbnfqG trigger"
          onBlur={[Function]}
          onClick={[Function]}
          size="large"
        >
          <div
            className="sc-VigVT ceQhNn"
          >
            <span
              className="sc-gZMcBi bvGPfj"
            >
              <span
                className="sc-dnqmqq hPNZvv"
              />
              <span
                className="sc-dnqmqq gsxYuK"
              />
              <span
                className="sc-dnqmqq jrdsWP"
              />
            </span>
          </div>
          <div
            className="sc-jTzLTM hoEajO"
          >
            All
          </div>
          <i
            className="fas fa-caret-down"
          />
        </button>
      </div>
    </div>
    <div
      className="sc-kpOJdX iVjlhL"
    >
      <div
        className="sc-htoDjs ixiQr sc-healthselector"
      >
        <button
          className="sc-bxivhb sc-gqjmRU fdNFQC trigger"
          onBlur={[Function]}
          onClick={[Function]}
          size="larger"
        >
          <div
            className="sc-VigVT ceQhNn"
          >
            <span
              className="sc-gZMcBi bvGPfj"
            >
              <span
                className="sc-dnqmqq hPNZvv"
              />
              <span
                className="sc-dnqmqq gsxYuK"
              />
              <span
                className="sc-dnqmqq jrdsWP"
              />
            </span>
          </div>
          <div
            className="sc-jTzLTM hoEajO"
          >
            All
          </div>
          <i
            className="fas fa-caret-down"
          />
        </button>
      </div>
    </div>
  </div>
  <h3
    className="sc-kAzzGY cVbXJ"
  >
    With custom labels (Triggering StoryBook actions)
  </h3>
  <div
    className="sc-htoDjs ixiQr sc-healthselector"
  >
    <button
      className="sc-bxivhb sc-gqjmRU bGfYun trigger"
      onBlur={[Function]}
      onClick={[Function]}
      size="base"
    >
      <div
        className="sc-VigVT ceQhNn"
      >
        <span
          className="sc-gZMcBi bvGPfj"
        >
          <span
            className="sc-dnqmqq hPNZvv"
          />
          <span
            className="sc-dnqmqq gsxYuK"
          />
          <span
            className="sc-dnqmqq jrdsWP"
          />
        </span>
      </div>
      <div
        className="sc-jTzLTM hoEajO"
      >
        Custom1
      </div>
      <i
        className="fas fa-caret-down"
      />
    </button>
  </div>
</div>
`;
